README.md

```diff
@@ -61,6 +61,7 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 - [RDFminer](https://github.com/Wimmics/RDFminer) - Web application to automatically discover SHACL shapes representative of an RDF data graph, by Wimmics; `CECILL-C` license; `Java`.
 - [SHACL Discovery Service](https://github.com/AKSW/discover-shacl-shapes) - Example implementation of a discovery service for SHACL shapes/shape groups; `MIT` license; `PHP`.
 - [Shapes of You index](https://index.semanticscience.org/) - SPARQL queries, OWL/SKOS vocabularies, SHACL/ShEx shapes, indexed from public `git` repositories; `MIT` license; `Typescript`, `Python`.
+- [SHACL Play!](https://shacl-play.sparna.fr/play/shapes-catalog) - To see your shapes file listed here, add it to the [Shapes Catalog source file on Github](https://github.com/sparna-git/SHACL-Catalog/blob/master/shacl-catalog.ttl).
```

Comment:
change link name to "SHACL Play! Catalog"

Reply:
When making proposals like this in the future I would suggest using the add a suggestion button in the reviewer so you can propose a diff that the reviewee can commit directly. Helps speed up the process for everyone.

<img width="275" alt="Screenshot 2024-12-03 at 10 11 43" src="https://github.com/user-attachments/assets/dc64ab86-f3a3-421a-ab8e-391a92570dce">

Reply:
See https://github.com/w3c-cg/awesome-semantic-shapes/pull/14#discussion_r1867433354 for what this looks like.